samples/vision_edge/edge_container_predict/automl_vision_edge_container_predict.py

```diff
@@ -37,10 +37,10 @@ def preprocess_image(image_file_path, max_width, max_height):
     Args:
         image_file_path: Path to a local image for the prediction request.
         max_width: The max width for preprocessed images. The max width is 640
-            (1024) for AutoML Vision image classfiication (object detetion)
+            (1024) for AutoML Vision image classfication (object detetion)
```

Comment:
NIT: I think that the branding calls for initial capitals: AutoML Vision Image Classification. See here:
https://cloud.google.com/vision/overview/docs#automl-vision

Reply:
Also, "detection" is spelled wrong and we present Image Detection and Image Classification as two separate objectives.

Reply:
done. Thanks for the comments!